src/shopify.js

```diff
@@ -41,8 +42,9 @@ const Shopify = {
    * @param {String} configAttrs.appId The app whose listings the client will be
    * using. If you are just modifying a buy button, the buy-button's app id is
    * 6. Otherwise, obtain the app id of the app you're modifying or extending.
-   * @param {String} configAttrs.myShopifyDomain You shop's `myshopify.com`
+   * @param {String} configAttrs.myShopifyDomain [DEPRACATED] You shop's `myshopify.com`
```

Comment:
There's actually an API for `[DEPRECATED]`

![image](https://cloud.githubusercontent.com/assets/568174/16462941/9a692cd0-3e01-11e6-8f8b-76ce28d94cd4.png)

From [here](http://yui.github.io/yuidoc/syntax/)